base/bucket_n1ql.go

```diff
@@ -397,6 +397,10 @@ func isIndexerError(err error) bool {
 		return true
 	}
 
+	if strings.Contains(err.Error(), "Build Already In Progress") {
```

Comment:
It looks like we're partially covering this particular error already, with the IsIndexerRetryBuildError above, but only using it when we issue the actual BUILD.  It feels like we should just invoke that from here, instead of adding another string match?

Reply:
Oops - didn't even see that. I'll update to use the existing function.